apps/backend/src/app/api/v1/users/crud.tsx

```diff
@@ -158,6 +167,11 @@ export const usersCrudHandlers = createLazyProxy(() => createCrudHandlers(usersC
         }
       }
 
+      // If the selected_team_id is present and we reach here that means user does exist. Hence, we have this check.
+      if (!data.selected_team_id) {
+        await ensureUserExist(tx, { projectId: auth.project.id, userId: params.user_id });
```

Comment:
We can move this check to the beginning and remove the condition so it is easier to follow